java/arcs/core/host/ParticleSpec.kt

```diff
@@ -0,0 +1,9 @@
+package arcs.core.host
+
+/**
+ * A [ParticleSpec] consists of the information neccessary to instantiate a particle
+ * when starting an arc.
+ * [manifestName] is human readable of the Particle in the recipe.
+ * [location] is either a fully qualified Java class name, or a filesystem path.
+ */
+data class ParticleSpec(val manifestName: String, val location: String)
```

Comment:
what is `manifestName`? in the ts code this would be the name of the .arcs file. does this refer to the name of the particle, or the name of the file?

Reply:
is says so in the Java doc comment above [manifestName]